src/Microsoft.Tye.Hosting/TransformProjectsIntoContainers.cs

```diff
@@ -86,7 +86,9 @@ private async Task TransformProjectToContainer(Service service, ProjectRunInfo p
 
         private static string DetermineContainerImage(ProjectRunInfo project)
         {
-            return $"mcr.microsoft.com/dotnet/core/sdk:{project.TargetFrameworkVersion}";
```

Comment:
This is gonna conflict with mah changes

Reply:
Let's discuss, to implement what you're doing and this feature. both important.